akka-cluster/src/main/protobuf/ClusterMessages.proto

```diff
@@ -97,14 +97,28 @@ message CompatibleConfig {
  ****************************************/
 
 /**
- * Heartbeat
- * Sends an Address
+ * Prior to version 2.5.23
+ * Heartbeat sends an Address
+ * Version 2.5.23 can deserialize this message but does not send it.
+ * FIXME: Issue for serializing this message
  */
+message Heartbeat {
+  required Address from = 1;
+  optional int64 sequenceNr = 2;
+  optional int64 sendTime = 3;
```

Comment:
`sint64` since nanoTime is equally likely to be negative? https://developers.google.com/protocol-buffers/docs/encoding#signed-integers